packages/actionmenu/src/react/arrow.tsx

```diff
@@ -2,7 +2,7 @@ import { HTMLPropsFor } from '@pluralsight/ps-design-system-util'
 import { css } from 'glamor'
 import React from 'react'
 
-import stylesheet from '../css'
+import stylesheet from '../css/index.js'
```

Comment:
@jaketrent did importing `.ts` break tree shaking?

Reply:
No, I don't think so. But I'm not sure why you ask, so maybe there's something I'm missing. Is there something you know about that we should verify related to tree shaking?

This change was made to provide explicit file extensions, required by esm, that TS allows but will not rewrite for us in compilation. (see linked TS issues in the PR)

Reply:
Didn't read those it's clear now. Thanks!

Reply:
However it looks like that is what's causing the test failures.  🤷🏿‍♂️